basketball_reference_web_scraper/client.py

```diff
@@ -61,12 +61,13 @@ def player_box_scores(day, month, year, output_type=None, output_file_path=None,
 
 
 def regular_season_player_box_scores(player_identifier, season_end_year, output_type=None, output_file_path=None,
-                                     output_write_option=None, json_options=None):
+                                     output_write_option=None, json_options=None, include_dnp=False):
```

Comment:
Nit, but mind calling this something like `include_inactive_games`?

A couple reasons why (in my mind) this might be slightly nicer:

1. `dnp` is an acronym, and while most basketball people know what DNP stands for, there might be some that don't
1. DNP seems like it's a pretty specific player designation - for example, Basketball Reference has other specific types of "inactive" player designations

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/8136030/94316818-1224ee00-ff53-11ea-8481-eb6a0096483f.png">

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/8136030/94316903-3f719c00-ff53-11ea-8770-0aa6e3f89a5d.png">

It's unfortunate that `inactive` itself is a player designation - but the term "inactive" maybe encapsulates the concept of a player being "out" or "unavailable to play".

I'm open to other suggestions.

Reply:
Doesn't matter to me, will update to `include_inactive_games`